src/Components/Menu/Menu.js

```diff
@@ -0,0 +1,40 @@
+import "./Menu.scss";
+
+const Menu = ({ menuItems, searchTerm }) => {
+  let dataToDisplay = menuItems;
+
+  if (searchTerm) {
+    dataToDisplay = menuItems.filter((item) => {
+      const { name, shortDescription } = item;
+      return (
+        name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+        shortDescription.toLowerCase().includes(searchTerm.toLowerCase())
```

Comment:
Minor comment but calling `searchTerm.toLowerCase()` twice is a bit inefficient. Lowercasing takes `O(n)` time so we'd rather not do it twice if it's not necessary. As an alternative you could save the lowercased search term to a variable before your or statement.